src/main/oAuthClient.ts

```diff
@@ -639,12 +639,22 @@ export default class OAuthClient {
     return Promise.resolve()
   }
 
-  private loginWithVerificationCode(params: VerifyPasswordlessParams, auth: AuthOptions = {}): void {
+  private loginWithVerificationCode(params: VerifyPasswordlessParams, auth: AuthOptions = {}): Promise<void> {
```

Comment:
@sachaMemmir5  How can the `AuthResult` be obtained if using `useWebMessage: true`?